### ISSUE TITLE
AvCheckbox always select all the options

### ISSUE THREAD

**Issue by winter-sleep** [Availity/availity-reactstrap-validation#114]
Seems AvCheckboxGroup always select all the options, as long as I give at least 1 value (whatever the value is). 

This issue also happens in official example https://availity.github.io/availity-reactstrap-validation/components/avform/

You can try change the options in `locationQualities`. all the checkboxes will be selected even you only give one of the value e.g. 'awesome'.

```
locationQualities: [
        'beautiful',
        'awesome',
        'wonderful',
      ],
```

```
          <AvCheckboxGroup inline name="locationQualities" label="Location Qualities" required>
            <AvCheckbox label="Beautiful" value="beautiful" />
            <AvCheckbox label="Awesome" value="awesome" />
            <AvCheckbox label="Wonderful" value="wonderful" />
          </AvCheckboxGroup>
```

**Comment by macoto-liu** [comment]
the same issue, I'm using availity-reactstrap-validation 2.5.0

**Comment by TheSharpieOne** [comment]
To be clear, this happens when using the `model` prop on `AvForm` to initialize the value of the field(s) as well as when providing `defaultValue` and even `value` (`value` probably should also "lock" the values of the checkboxes to the value as a normal input would do with it's value).

model: https://stackblitz.com/edit/reactstrap-validation-v2-hlzesa?file=Example.js
defaultValue: https://stackblitz.com/edit/reactstrap-validation-v2-spxizj?file=Example.js

**Comment by raheelriax** [comment]
> AvCheckbox.prototype.isDefaultChecked = function isDefaultChecked(valueArr) {
> var _this2 = this;
> 
> ```
> /*
> return Array.isArray(valueArr) && valueArr.length > 0 && valueArr.filter(function (item) {
>   return item === _this2.props.value;
> });
> */
> ```
> let dChecked = false;
> if(Array.isArray(valueArr) && valueArr.length > 0){
> for(let i = 0; i < valueArr.length; i ++){
> let item = valueArr[i];
> if(item === _this2.props.value){
> dChecked = true;
> break;
> }
> }
> }
> return dChecked;
> };

@odlar make a PR for it

**Comment by maciejzukowski** [comment]
#120

**Comment by GoPro16** [comment]
Closed via #120
